src/riscv/lib/src/jit/builder/outcome_map.rs

```diff
@@ -26,6 +26,19 @@ use std::ops::Range;
 
 use crate::jit::builder::instr_map::InstrId;
 
+/// Probability of taking an outcome of a particular instruction.
+///
+/// As detailed in RISC-V Control Transfer Instructions specification (2.5),
+/// backward-branches should be treated as likely taken, while forward-branches
+/// should be treated as unlikely taken. Also, exception handlers should be treated
+/// as unlikely taken (apart from ECall, which is definite).
```

Comment:
`ECall` is not the only one? There's `Unknown -> IllegalInstruction` and `ForceFetchRun -> exception` too


```suggestion
/// as likely not-taken (except for a few instructions, such as `ECall`, which are guaranteed to 
/// result in an exception).
```